virotour/lib/src/tour/tour_list_view.dart

```diff
@@ -56,14 +56,65 @@ class _TourListViewState extends State<TourListView> {
     return Scaffold(
       appBar: AppBar(
         title: const Text('Tours'),
-        actions: [
-          IconButton(
+        actions: [                  //wheel_menu action start
+          PopupMenuButton<int>(
+            color: Colors.lightBlue[600],
             icon: const Icon(Icons.settings),
-            onPressed: () {
-              Navigator.restorablePushNamed(context, SettingsView.routeName);
-            },
-          ),
-        ],
+            onSelected: (item) => onSelected(context, item),
```

Comment:
Please make the wheel menu options visible on all views rather than just List of Tours view.